app/models/client.rb

```diff
@@ -35,16 +35,17 @@ def self.delegated_intake_attributes
   end
 
   delegate *delegated_intake_attributes, to: :intake
-
-  scope :assigned_to, ->(user) { joins(:tax_returns).where({ tax_returns: { assigned_user_id: user } }) }
+  scope :after_consent, -> { distinct.joins(:tax_returns).merge(TaxReturn.where("status > 100")) }
+  scope :assigned_to, ->(user) { joins(:tax_returns).where({ tax_returns: { assigned_user_id: user } }).distinct }
 
   scope :delegated_order, ->(column, direction) do
     raise ArgumentError, "column and direction are required" if !column || !direction
 
     if delegated_intake_attributes.include? column.to_sym
-      joins(:intake).merge(Intake.order(Hash[column, direction]))
+      select = delegated_intake_attributes.inject(["clients.*"]) { |arr, attr| arr << ("intakes." + attr.to_s) }
```

Comment:
In order to use "distinct", we need to include the intake properties in delegated_intake_attributes in our select. We _could_ limit it to just the column we're passing in, but I was worried it might trigger a Hakiri violation because we'd be creating a query from user-provided data.